slides/rights_and_licenses.md

```diff
@@ -0,0 +1,95 @@
+---
+title:  Rights and Licenses
+author: Anni Järvenpää
+date:   2023-06-20
+lang:   en
+---
+
+
+# In This Session
+
+- What kinds of licenses can resources offered via the Language Bank have?
+- How to access the resources?
+- What you can do with them?
+- What about very restricted data?
+
+
+# License Categories
+
+![Icon for PUB resources](img/pub.png){width=30%}
+![Icon for ACA resources](img/aca.png){width=30%}
+![Icon for RES resources](img/res.png){width=30%}
+
+- Broad categories
+- Outline the process you need to follow to get access to the resource
+- Further license conditions can and often will apply!
+
+
+# PUB Resources
+
+<img src="img/pub.png" style="display:inline-block;position:absolute;left:500px;top:-20px;height:220px;">
+
+- Publicly distributed resources
+- No login required
+- Other conditions may apply:
+  - attributing the original creator
+  - sharing your creations with same license
+  - non-commercial use only
+  - ...
+
+
+# ACA Resources
+
+<img src="img/aca.png" style="display:inline-block;position:absolute;left:500px;top:-20px;height:220px;">
+
+- Resources that are available for academic users
+- Login required
+- Other conditions may apply:
+  - attributing the original creator
+  - sharing your creations with same license
+  - non-commercial use only
+  - ...
+
+# RES Resources
+
+<img src="img/res.png" style="display:inline-block;position:absolute;left:500px;top:-20px;height:220px;">
+
+- The most restricted license category
+- Often contain copyrighted material and/or personally identifiable information
+- Accessing a resource requires filling an application
+  - Authenticated applicant
+  - Explicit agreement to license terms
+  - Research plan
+  - Possibly other requirements too, e.g. specific affiliation or submitting a privacy statement
+
+
+# LBR
+
+<div class="column" style="float:left">
+- [lbr.csc.fi](https://lbr.csc.fi)
+- Service in which you can apply for access to RES licensed resources
+- Fields vary depending on the resource, at least purpose of use in all forms
+</div>
+<img src="img/lbr-fields.png" style="display:inline-block;position:absolute;max-width:none;max-height:none;height:1050px;top:10px">
+
+# LBR: privacy statement
+
```

Comment:
Maybe say this has an explicit action item: eg. you have to fill a privacy statement to access RES resources